### ISSUE TITLE
greenwood website home page logo fade not working with internet explorer 11 (syntax error?)

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#198]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
Loading the Greenwood website in IE11 and the home page loads, but the fading text doesn't show and the button color is green not blue (like in #196 ).
<img width="1220" alt="ie-11-serve-home-page-no-animation" src="https://user-images.githubusercontent.com/895923/65085137-4bf40980-d97b-11e9-81bb-23cc828e9a92.png">

Otherwise the site is working pretty good!

## Details
Maybe something to do with animation frames?  Not sure if that's a different issue from the syntax error?
<img width="1257" alt="ie-11-serve-home-page-syntax-error" src="https://user-images.githubusercontent.com/895923/65085163-5f06d980-d97b-11e9-96d0-87f100da2e49.png">

**Comment by thescientist13** [comment]
Wondering if [es5 adapter](https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs#custom-elements-es5-adapterjs) would be needed for this?

Or do we need a PostCSS plugin for this for CSS `@keyframe` / `animation`?

**Comment by thescientist13** [comment]
Trying a few things to fix the loading error, no luck so far
- https://stackoverflow.com/a/52264800/417806
- https://github.com/Polymer/lit-html/issues/516#issuecomment-475261804

Created [a branch](https://github.com/ProjectEvergreen/greenwood/tree/bug/issue-198-ie11-loading-syntax-error) to try some of these things out with this and other various attempts

**Comment by thescientist13** [comment]
oh, right.  [this](https://github.com/ProjectEvergreen/greenwood/blob/bug/issue-198-ie11-loading-syntax-error/packages/cli/src/config/webpack.config.common.js#L88) should probably use `process.cwd()`, not `__dirname`.  🤦‍♂

**Comment by thescientist13** [comment]
So digging deeper into this, it looks like _.babelrc_ isn't even getting loaded, likely because **babel-loader** is looking for the file in the root.   🤦‍♂ 

Once I added it as an option
```javascript
module: {
  rules: [{
    test: /\.js$/,
    loader: 'babel-loader',
    options: {
      configFile: path.join(__dirname, 'babel.config.js'),
      include: [/node_modules/]
    }
  }, {
```

things started ~~working~~ breaking, so at least something is happening now, but will need to go down the rabbit hole a bit on this though, but definitely should git it fixed.

Will also probably want to look into checking _.browserslistrc_ is also configured correctly as well.

**Comment by thescientist13** [comment]
OK, got babel configuration working, but still seeing an issue in IE11 with `class` not getting recognized
<img width="1026" alt="ie-11-transpilation-class" src="https://user-images.githubusercontent.com/895923/68726704-581dd100-0590-11ea-9250-337313a32e9b.png">

Made an issue for tracking babel configuration specifically, removing `P0` from this ticket
https://github.com/ProjectEvergreen/greenwood/issues/256

**Comment by thescientist13** [comment]
Will need to enable process of _node_modules_, but this might mean it makes sense to do #224 instead...

**Comment by aholtzman** [comment]
Considering the audience for this site is primarily developers and others within the technology realm it seems incredibly unlikely that anyone visiting the site with IE 11 would be doing so intentionally. I vote to not worry about supporting IE and to close Thai and any related issues.

**Comment by thescientist13** [comment]
Hmm, I certainly see your point regarding maintaining the website, but I wonder if the real world use cases out there support it though?  I feel like especially in corporate / enterprise and some government, IE11 is still a meaningful consideration in regards to of their user's browser demographics and so I feel like it's still worth tracking #224 at least.  (I guess i was mostly keeping this open to keep us honest about the limitations of Greenwood.)

Basically, I would like to piggy back of our polyfills plugin and add support for (opt-in) [differential loading](https://web.dev/serve-modern-code-to-modern-browsers/) so that (maybe in combination with our Babel plugin) a user could still build with modern standards and WCs, but still have the option to operate in an ES5 environment.  WebComponents polyfills provide an [ES5 adapter](https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs#custom-elements-es5-adapterjs) which I think we would want to leverage in this case.

----

TL;DR I agree for the website we can drop IE11 (close this issue), but I think providing a differential loading mechanism is still useful, so still keep #224 open,  though at a lower priority unless someone asks for it perhaps?  When we do add it, that's when we can use the website as an opportunity for dog fooding.

Thoughts?
